store/tikv/gcworker/gc_worker_test.go

```diff
@@ -188,6 +188,27 @@ func (s *testGCWorkerSuite) TestGetOracleTime(c *C) {
 	s.timeEqual(c, t2, t1.Add(time.Second*10), time.Millisecond*10)
 }
 
+func (s *testGCWorkerSuite) TestMinStartTS(c *C) {
+	spkv := s.store.GetSafePointKV()
+	err := spkv.Put(fmt.Sprintf("%s/%s", domain.ServerMinStartTSPath, "a"), strconv.FormatUint(math.MaxUint64, 10))
+	c.Assert(err, IsNil)
+	now := time.Now()
+	sp := s.gcWorker.calSafePointByMinStartTS(now)
+	c.Assert(sp.Second(), Equals, now.Second())
+	err = spkv.Put(fmt.Sprintf("%s/%s", domain.ServerMinStartTSPath, "a"), "0")
+	c.Assert(err, IsNil)
+	sp = s.gcWorker.calSafePointByMinStartTS(now)
+	zeroTime := time.Unix(0, oracle.ExtractPhysical(0)*1e6)
+	c.Assert(sp, Equals, zeroTime)
+
+	err = spkv.Put(fmt.Sprintf("%s/%s", domain.ServerMinStartTSPath, "a"), "0")
+	c.Assert(err, IsNil)
+	err = spkv.Put(fmt.Sprintf("%s/%s", domain.ServerMinStartTSPath, "b"), "1")
+	c.Assert(err, IsNil)
+	sp = s.gcWorker.calSafePointByMinStartTS(now)
+	c.Assert(sp, Equals, zeroTime)
```

Comment:
I think you may at least need to add a case that saves a normal time to etcd, to check if `calSafePointByMinStartTs` parses the time correctly.

Reply:
Let's add some integration tests.